Sources/SQL/SQLLiteral.swift

```diff
@@ -2,11 +2,12 @@ public protocol SQLLiteral: SQLSerializable {
     associatedtype DefaultLiteral: SQLDefaultLiteral
     associatedtype BoolLiteral: SQLBoolLiteral
     static func string(_ string: String) -> Self
+    static func rawString(_ string: String) -> Self
```

Comment:
I like this change, but I wonder if we should call this something like `keyword` or `sql`? That could potentially be a more obvious difference than `rawString` vs. `string`.

Reply:
Hi, I think SQLIdentifier support isKeyword is better. what do you think?